src/test_utils.rs

```diff
@@ -624,14 +626,36 @@ impl TestContext {
         res
     }
 
+    #[allow(clippy::unused)]
+    pub async fn golden_test_chat(&self, chat_id: ChatId, filename: &str) {
+        let filename = "test-data/golden/".to_owned() + filename;
```

Comment:
I'm sort of used to seeing this as `Path` with a `.join()`, but whatever, it's nitpicking.

Reply:
:heavy_check_mark: